requirements/dev.txt

```diff
@@ -1,3 +1,4 @@
 -r ci.txt
 -r towncrier.txt
 cherry_picker==1.3.2; python_version>="3.6"
+freezegun==0.3.12
```

Comment:
This should be added to ci.txt, not here. Appveyor breaks because this is the wrong location for the dependency.

Reply:
Moved.